cli/src/base/StakingCommandBase.ts

```diff
@@ -0,0 +1,95 @@
+import ExitCodes from '../ExitCodes'
+import { AccountId } from '@joystream/types/common'
+import { StakingLedger } from '@polkadot/types/interfaces'
+import AccountsCommandBase from './AccountsCommandBase'
+import { formatBalance } from '@polkadot/util'
+
+export default abstract class StakingCommandBase extends AccountsCommandBase {
+  private async availableControllers(): Promise<[AccountId, StakingLedger][]> {
+    const controllers = await this.getStakingLedgers()
+    return controllers
+      .filter(([key, value]) => value.isSome && this.isKeyAvailable(key))
+      .map(([key, value]) => [key, value.unwrap()])
+  }
+
+  private async getStakingLedgers() {
+    const controllers = await this.getApi().allStakingLedgers()
+    return controllers
+  }
+
+  async getController(): Promise<string> {
+    const availableControllers = await this.availableControllers()
+    const selectedValidator = await this.promptForController('No controller input', availableControllers)
```

Comment:
```suggestion
    const selectedValidator = await this.promptForController(undefined, availableControllers)
```
I think the default prompt message (`Choose your controller account`) is more clear, but it's not a big issue.